### ISSUE TITLE
Lorre should not die when the Tezos node cannot be contacted

### ISSUE THREAD

**Issue by vishakh** [Cryptonomic/Conseil#269]
**Background**

When the Tezos node and Lorre are deployed at the same time, it takes the Tezos node a few minutes to start responding to queries. However, Lorre tries to query Tezos as soon as it starts up. Since Lorre does not get a response from Tezos, it dies.

**Proposed Solution**

Lorre's behaviour should be changed so that, instead of dying, it keeps trying to contact Tezos. Furthermore, it should leave log output indicating that the Tezos node cannot be queried.

**Steps for testing**

1. Give Lorre an incorrect Tezos URL and ensure it still keeps running. 
2. Start a local Tezos node and start Lorre at the same time and ensure Lorre can eventually start talking to Tezos.

For step 2, @leenpaws can provide any required information.

**Comment by piotrkosecki** [comment]
I've looked for the reasonable solution but it occurred to me that it is already implemented. 
I've tested it with the local Tezos node starting/stopping it manually. I believe @ivanopagano already implemented mechanism for skipping processing failures and retrying. It uses environment variable `LORRE_FAILURE_IGNORE`. If it is set to `true` in case of failure it will wait for `sleepInterval` amount of time which is defined in config to try again to connect to Tezos node. @leenpaws have you tried it? From what I've been testing it should solve this problem.
